tests/resources/image10x11x12x13-extra-stuff.mhd

```diff
@@ -4,7 +4,7 @@ BinaryData = True
 BinaryDataByteOrderMSB = False
 CompressedData = True
 CompressedDataSize = 39
-TransformMatrix = 1 0 0 0 0 1 0 0 0 0 1 0 0 0 0 1
+TransformMatrix = 1 0 0 0 0 1 0 0 1 0 1 0 0 0 0 1
```

Comment:
Did you add this for testing purposes or did the output change? Is this what you mean with "I am yet unsure where in the (Simple)ITK internals things go wrong when providing a Directions from a specific TransformMatrix. Possibly an interaction with the AnatomicalOrientation..."

We should understand this and be careful here as it could have consequences.

Reply:
uh oh.... if that changed, that indeed seems like a significant change.... Orientation matrices should not be touched and this particular change looks a lot like a shear-factor being introduced? That is possible but is really unusual.

Reply:
Added for testing purposes. After some back-and-forth I found out this was the minimum matrix I could generate to reproduce the original behaviour. I have a vague theory related to summation float precision, oblique acquisition angles, and many DICOM -> one MHA. I'll spend some time investigating it further.

Reply:
See: https://github.com/DIAGNijmegen/rse-panimg/issues/44#issuecomment-921870930